### ISSUE TITLE
2.5.3 Label in Name - case insensitve?

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#3812]
I'm going to assume (based on the described use cases/benefits) that 2.5.3 is case insensitive. For instance, I'd assume the following passes:

```
<button aria-label="Delete">DELETE</button>
```

If others agree, suggest this should be clarified/mentioned in the understanding document (don't think it currently does, unless I missed)

https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html

**Comment by giacomo-petri** [comment]
I’m curious if it makes a difference whether an acronym is used in uppercase vs lowercase. 

Let's consider:
- "ADD" as a word in the context of "add to bag":
`<button aria-label="add">ADD</button>`
No issues
- "ADD" as an acronym for "Attention Deficit Disorder":
`<a href="..." aria-label="add">ADD</a>`
Could there be any issue here?

I know that VoiceOver, for instance, tends to read uppercase acronyms as individual letters. However, I’m unsure how speech recognition technologies handle this. Do they recognize acronyms letter by letter, or do they interpret the entire word? 

In the context of "Attention Deficit Disorder," I wouldn't expect someone to pronounce "add" as a word. Instead, I would anticipate it being spelled out as "A," "D," "D."

Does it make a difference for speech recognition technology users if you use:

`<a href="..." aria-label="add">ADD</a>`
vs
`<a href="..." aria-label="ADD">ADD</a>`

?

**Comment by filippo-zorzi** [comment]
Voice Control (with Chrome, for what it matters) doesn't distinguish between "add" and "ADD".

https://github.com/user-attachments/assets/dcbc1e9c-ff3b-48f9-acbd-de73348690ff

"A.D.D." has to be spelled out but this is not in the focus of this issue.

**Comment by giacomo-petri** [comment]
Another relevant example is when uppercase and lowercase letters convey different meanings. For instance, in chemistry, "CO" represents carbon monoxide, while "Co" stands for cobalt. "CO" is read letter by letter as "C–O", whereas "Co" is pronounced as a single syllable, "Co" (at least in Italian).

Thus, although in most cases uppercase and lowercase make no difference, it is still important to include a note explaining that, when the specific letter case is meaningful, the accessible name must reflect the visual representation.

If we all agree, I'll take care of it.

**Comment by patrickhlauke** [comment]
> Another relevant example is when uppercase and lowercase letters convey different meanings. For instance, in chemistry, "CO" represents carbon monoxide, while "Co" stands for cobalt. "CO" is read letter by letter as "C–O", whereas "Co" is pronounced as a single syllable, "Co" (at least in Italian).
> 
> Thus, although in most cases uppercase and lowercase make no difference, it is still important to include a note explaining that, when the specific letter case is meaningful, the accessible name must reflect the visual representation.
> 
> If we all agree, I'll take care of it.

But from an actual practical level - the whole "users with speech recognition / voice control" scenario that 2.5.3 aims to address ... this distinction is perhaps irrelevant? Does voice control even distinguish between a user spelling things out versus saying them as if they were a word? (this is where the well-meaning intention of 2.5.3 probably hits against actual wishful thinking, perhaps)

EDIT: ah, I see Filippo made a similar point last year, with actual example https://github.com/w3c/wcag/issues/3812#issuecomment-2289090993

Sure, there will be side effects for other scenarios, like actual SR users...but that doesn't, i think, fall under what 2.5.3 actually tries to address, strictly? Maybe a more general best practice note, mentioning that although for 2.5.3 normatively you can treat it as case-insensitive, authors should strive to keep the case for the accName the same as for the visible text.

**Comment by alastc** [comment]
TF meeting:
Recommend adding something to the Understanding doc, saying it is not case-sensitive. Do a little checking with Dragon / VoiceControl.

**Comment by patrickhlauke** [comment]
I verified that, at least on iOS Voice Control, case makes no difference. in addition, whether the user spells out things (like "A ... D ... D") or says them as a single word (like "add") the results are the same. I'll draft a little note for 2.5.3 to summarise what i think is the outcome of this whole discussion
